src/main/sentry.js

```diff
@@ -29,6 +31,11 @@ Sentry.init({
       service: service,
       walletType: identifier
     };
+    const err = event.exception.values[0].value;
+    if (knownErrors[err]) {
```

Comment:
Use the same check for `WalletErrorHandler.js` so it doesn't have to be 1:1 match for these errors